digiroad2-oracle/src/main/resources/db/migration/V131__add_required_attributes.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE PROJECT_LINK ADD ELY NUMBER;
```

Comment:
Can't replace a migration file that is already in master

Reply:
Ok, i will change it for V132